### PR TITLE
[17.01] Migrate galaxy_utils to a wheel.

### PR DESCRIPTION
**Pull request by jmchilton**
Alternative approach to #3542 which was not future proof with Conda dependencies. I think this should be now.

xref #3364 #2994 http://dev.list.galaxyproject.org/Tools-using-Galaxy-s-Python-library-td4670472.html

### CODE REVIEW

File: lib/galaxy/dependencies/pinned-requirements.txt [hunk 1]
```diff
@@ -30,6 +30,7 @@ Parsley==1.3
 six==1.10.0
 Whoosh==2.7.4
 testfixtures==4.10.0
+galaxy_sequence_utils=1.0.2
```
Comment (mvdbeek) [inline lib/galaxy/dependencies/pinned-requirements.txt]: aha, someone's been doing conda ;)

Comment (nsoranzo) [reply lib/galaxy/dependencies/pinned-requirements.txt]: https://travis-ci.org/galaxyproject/galaxy/jobs/197897239#L164
```
Invalid requirement: 'galaxy_sequence_utils=1.0.2'
= is not a valid operator. Did you mean == ?
```

**Comment by nsoranzo** [comment]
You can also remove the lines containing `lib/galaxy_utils/` from `.ci/py3_sources.txt` and `.ci/flake8_lint_include_list.txt` .

**Comment by jmchilton** [comment]
Thanks all - I've fixed the .ci include lists and requirement version specifier. Lets see if the tests work any better now.

**Comment by jmchilton** [comment]
Opps I didn't notice that @martenson has already fixed the requirements and I forced push over his changes - sorry @martenson.

File: lib/galaxy/dependencies/requirements.txt [hunk 2]
```diff
@@ -30,6 +30,7 @@ nose
 Parsley
 six
 Whoosh
```
Comment (nsoranzo) [inline lib/galaxy/dependencies/requirements.txt:32]: Not connected to this PR, but it seems
```
testfixtures
```
is missing here.

**Comment by martenson** [comment]
@jmchilton lol, no trouble, just Travis is confused, I will cancel one job

**Comment by mvdbeek** [comment]
@galaxybot test this

**Pull request merged**